src/ipaddress.cpp

```diff
@@ -3,37 +3,66 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
I'm not seeing anything particularly wrong, but I feel as though the netmasking might be a little easier to understand if we added a static method along the lines of `BigIntIPv6Addr::fill()` or `BigIntIPv6Addr::netmask()` that just returns an IPv6 address with a netmask of the desired length.